xarray/core/rolling.py

```diff
@@ -234,12 +238,12 @@ def construct(
 
         Parameters
         ----------
-        window_dim: str or a mapping, optional
+        window_dim : str or mapping, optional
             A mapping from dimension name to the new window dimension names.
             Just a string can be used for 1d-rolling.
-        stride: integer or a mapping, optional
+        stride : int or mapping of int, optional
             Size of stride for the rolling window.
-        fill_value: optional. Default dtypes.NA
+        fill_value : default: dtypes.NA
```

Comment:
should we link to `dtypes.NA` somewhere, just like `scalar`. Maybe in our terminology page?